R/as-data-frame.R

```diff
@@ -1,34 +1,64 @@
-CrunchDataFrame <- function (dataset) {
+CrunchDataFrame <- function (dataset, order) {
```

Comment:
I agree, I wasn't fully happy with `order` anyway and 1 is a very valid thing to ignore. I've changed it to `selection` which at a minimum avoids 1, but might still not be the best name (open to other suggestions!)